packages/wonder-blocks-typography/src/util/styles.ts

```diff
@@ -1,154 +1,143 @@
 import {StyleSheet} from "aphrodite";
 import type {StyleDeclaration} from "aphrodite";
-
-const Regular = 400;
-const Bold = 700;
-const Black = 900;
-
-const mobile = "@media (max-width: 1023px)";
-const desktop = "@media (min-width: 1024px)";
+import {font, breakpoint} from "@khanacademy/wonder-blocks-tokens";
 
 const common = {
     display: "block",
 } as const;
 
-const SansFamily = 'Lato, "Noto Sans", sans-serif';
-// TODO(kevinb): Use Minion Pro here
-const SerifFamily = '"Noto Serif", serif';
-const InconsolataFamily = "Inconsolata, monospace";
-
 const styles: StyleDeclaration = StyleSheet.create({
     Title: {
         ...common,
-        fontFamily: SansFamily,
-        fontWeight: Black,
-        [desktop]: {
-            fontSize: 36,
-            lineHeight: "40px",
+        fontFamily: font.family.sans,
+        fontWeight: font.weight.black,
+        [breakpoint.mediaQuery.xl]: {
+            fontSize: font.size.xxxLarge,
+            lineHeight: font.lineHeight.xxxLarge,
         },
-        [mobile]: {
-            fontSize: 28,
-            lineHeight: "32px",
+        [breakpoint.mediaQuery.mdOrSmaller]: {
+            fontSize: font.size.xxLarge,
+            lineHeight: font.lineHeight.xxLarge,
         },
     },
     Tagline: {
         ...common,
-        fontFamily: SansFamily,
-        fontWeight: Regular,
-        fontSize: 20,
-        lineHeight: "24px",
+        fontFamily: font.family.sans,
+        fontWeight: font.weight.regular,
+        fontSize: font.size.large,
+        lineHeight: font.lineHeight.large,
     },
     HeadingLarge: {
         ...common,
-        fontFamily: SansFamily,
-        fontWeight: Bold,
-        [desktop]: {
-            fontSize: 28,
-            lineHeight: "32px",
+        fontFamily: font.family.sans,
+        fontWeight: font.weight.bold,
+        [breakpoint.mediaQuery.xl]: {
+            fontSize: font.size.xxLarge,
+            lineHeight: font.lineHeight.xxLarge,
         },
-        [mobile]: {
-            fontSize: 24,
-            lineHeight: "28px",
+        [breakpoint.mediaQuery.mdOrSmaller]: {
+            fontSize: font.size.xLarge,
+            lineHeight: font.lineHeight.xLarge,
         },
     },
     HeadingMedium: {
         ...common,
-        fontFamily: SansFamily,
-        fontWeight: Bold,
-        [desktop]: {
-            fontSize: 24,
-            lineHeight: "28px",
+        fontFamily: font.family.sans,
+        fontWeight: font.weight.bold,
+        [breakpoint.mediaQuery.xl]: {
+            fontSize: font.size.xLarge,
+            lineHeight: font.lineHeight.xLarge,
         },
-        [mobile]: {
-            fontSize: 22,
-            lineHeight: "26px",
+        [breakpoint.mediaQuery.mdOrSmaller]: {
+            fontSize: font.size.large,
```

Comment:
**Note:** `sizing.size_220` isn't part of our REM token scale, so I adjusted it to `font.size.large` (`20px`)

Reply:
I think it would be better to keep it as one off for now, because it would use the same value defined for `HeadingSmall` below, and this would make these to headings look the same in small/medium screens.

```suggestion
            fontSize: `22px`,
```